koku/api/report/aws/query_handler.py

```diff
@@ -83,8 +83,11 @@ def annotations(self):
             annotations['usage_units'] = Coalesce(self._mapper.usage_units_key, Value(units_fallback))
         # { query_param: database_field_name }
         fields = self._mapper.provider_map.get('annotations')
+        prefix_removed_parameters_list = map(lambda x: x if ':' not in x else x.split(':', maxsplit=1)[1],
+                                             self.parameters.get('group_by', {}).keys())
+        parameters_list = self.parameters.get('group_by', {}).keys()
         for q_param, db_field in fields.items():
-            if q_param in self.parameters.get('group_by', {}).keys():
+            if q_param in parameters_list or q_param in prefix_removed_parameters_list:
```

Comment:
I don't think checking both lists helps here. Should be `if q_param in prefix_removed_parameters_list`